cdap-api/src/main/java/co/cask/cdap/api/security/securestore/SecureStoreMetadata.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.security.securestore;
+
+import java.util.Map;
+
+/**
+ * Represents the metadata for the element stored in the secure store.
+ */
+public interface SecureStoreMetadata {
```

Comment:
Do you have more than one implementation of this class? This class seems like a POJO to me.